src/components/Amalgamation/AmalgamatingBusinesses.vue

```diff
@@ -486,11 +474,23 @@ export default class AmalgamatingBusinesses extends Mixins(AmalgamationMixin, Co
       return
     }
 
-    // Check for Legal API fetch issues.
+    // Check for business not in LEAR (Auth/Legal dbs).
+    if (business.authInfo?.status === 'NOT_FOUND') {
+      // Report error.
+      console.log('Missing auth info.')
+      this.showUnableToAddBusinessDialog()
+
+      // Hide spinner.
+      this.$root.$emit('showSpinner', false)
+
+      return
+    }
```

Comment:
"If Auth API reports Not Found then the business doesn't exist in LEAR. Report error."